src/api/app/jobs/configuration_write_to_backend_job.rb

```diff
@@ -0,0 +1,5 @@
+class ConfigurationWriteToBackendJob < ApplicationJob
+  def perform(configuration_id)
```

Comment:
Nope, if `queue_as` isn't specified then ActiveJob will allocate it to the "default" queue.